frontend/src/metabase/query_builder/components/LimitInput.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
I'm wondering if we can reuse an existing input here with some tweaks if needed

Reply:
I'm not sure if we want to. This new `LimitInput` as actually the same as `Input` was before, but it is used only in these 2 places, and it's very different compared to our standard `input` for some reason. I'd leave it as it is for now.